product_operating_unit/tests/test_product_operating_unit.py

```diff
@@ -17,7 +17,8 @@ def setUp(self):
         # groups
         self.group_stock_user = self.env.ref('stock.group_stock_user')
         # Main Operating Unit
-        self.ou1 = self.env.ref('operating_unit.main_operating_unit')
```

Comment:
Why can't we use main_operating_unit when this module depends on operating_unit and it's defined there?

Reply:
We can use it, but it complicates making our tests run when other modules (which also use that OU) are installed. By using our own demo data we control which OU we use, and we are more independent from other modules.

Reply:
Ok, then leave it more independent ;-)